src/sonic-config-engine/data/l2switch.j2

```diff
@@ -6,8 +6,11 @@
         {%- if ns.firstPrinted %},{% endif %}
 
         "{{ key }}": {
-            "alias": "{{ value.alias }}",
-            "lanes": "{{ value.lanes }}",
+            {%- for keyPort,valuePort in value.items() %}
```

Comment:
>keyPort [](start = 20, length = 7)

If keyPort == 'admin_status', is it a problem?

Reply:
Added check to resolve the scenario with `admin_status` exists.